### ISSUE TITLE
iFrame accessibility 

### ISSUE THREAD

**Issue by CarolineRennie** [comicrelief/campaign#1406]
The current method of embedding forms and videos on the site via the CMS is not accessible. Below is the issue and proposed solution from DAC:

The YouTube video player is embedded in a frame that doesn’t have a title attribute. All <frame> or <iframe> elements must have a title that describes their contents to screen reader users. A frame missing a title attribute is also present on the ‘contact us’ page.
Code Ref(s):

<iframe src="https://www.youtube.com/embed/OR79o6dNDwQ?autoplay=1" width="100%" height="100%" frameborder="0" nuan_newframe="true" style="width: 100%; height: 100%; max-width: 800px; max-height: 500px;"></iframe>
Solution:
Ensure that all iframe and frame elements contain a unique and non-empty title attribute.
For example:
<iframe id="fillform-frame-1" class="achieveforms-iframe" title=”James Cracknell’s Epic Challenge - YouTube”>
</iframe>
